### PR TITLE
1 crud functionality books service

### PR DESCRIPTION
**Pull request by alexzubkoff**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        return Book.objects.filter(user=user).order_by("title")
+
+
+class BookListCreate(generics.ListCreateAPIView):
+    serializer_class = BookSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        user = self.request.user
+        return Book.objects.filter(user=user).order_by("title")
```
Comment (mltdll) [inline api/views.py]: Why do you need both of these views?

File: api/views.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class BookRetrieveUpdateDestroy(generics.RetrieveUpdateDestroyAPIView):
+    serializer_class = BookSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        user = self.request.user
+
+        return Book.objects.filter(user=user)
```
Comment (mltdll) [inline api/views.py]: Don't forget to add a newline at the end of the file.

**Review by mltdll — CHANGES_REQUESTED** [review]
I don't think it was necessary to create a separate `api` app. It seems to make more sense to just implement all the views and serializer in the `book`.
By the way, you should also resolve conflicts.

File: api/views.py (hunk 1 above)
Comment (alexzubkoff) [reply api/views.py]: I think it is correct if read all books list can any, but create only authenticated

Comment (mltdll) [reply api/views.py]: Then I'd suggest using `CreateAPIView` instead of `ListCreateAPIView`.

File: book/serializers.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    class Meta:
+        model = Book
+        fields = [
+            "id",
+            "title",
+            "author",
+            "cover",
+            "inventory",
+            "daily_fee"
+        ]
```
Comment (mltdll) [inline book/serializers.py]: You deleted the trailing blank line here. :)

**Review by mltdll — APPROVED** [review]

**Review by olia-trofymchuk — APPROVED** [review]

**Pull request merged**